docs/adr/0003-workspace-support-vscode-extension.md

```diff
@@ -0,0 +1,61 @@
+# Workspace support in the Inmanta vs-code extension
+
+* Status: accepted
+* Deciders: [Sander, Hugo]
```

Comment:
@sanderr I included you because you helped me quite a lot fleshing out the desired behaviour, should I include anyone else ?